src/patternfly/components/Pagination/pagination.scss

```diff
@@ -2,6 +2,10 @@
   // Base
   --pf-c-pagination--FontSize: var(--pf-global--FontSize--sm);
   --pf-c-pagination--child--MarginRight: var(--pf-global--spacer--lg);
+  --pf-c-pagination--m-footer--PaddingTop: var(--pf-global--spacer--md);
```

Comment:
Should the footer pagination be in a toolbar, so the spacing can just come from the toolbar in both instances and we don't have to match it here?

Reply:
Yes good idea!